package/src/components/Attachment/__tests__/Gallery.test.js

```diff
@@ -230,4 +230,34 @@ describe('Gallery', () => {
       expect(queryAllByTestId('gallery-column-1-item-1').length).toBe(1);
     });
   });
+
+  it('should render a loading indicator and when successful render the image', async () => {
+    const image1 = generateImageAttachment({
+      original_height: 300,
+      original_width: 600,
+    });
+
+    const component = await getComponent([image1]);
+    const { getByA11yLabel, getByAccessibilityHint } = render(component);
+
+    fireEvent(getByA11yLabel('gallery-image'), 'onLoadStart');
+    expect(getByAccessibilityHint('image-loading')).toBeTruthy();
+
+    fireEvent(getByA11yLabel('gallery-image'), 'onLoadFiniah');
+
+    expect(getByA11yLabel('gallery-image')).toBeTruthy();
```

Comment:
What is the idea behind this expect? `getByA11yLabel('gallery-image')` was always in the tree.

Please expect that the `(getByAccessibilityHint('image-loading')` disappears